tests/path.rs

```diff
@@ -301,9 +300,8 @@ fn no_rebuild_dependency() {
     // This time we shouldn't compile bar
     assert_that(p.cargo("build"),
                 execs().with_stdout(""));
-    p.root().move_into_the_past();
+    p.root().move_into_the_future();
 
-    p.build(); // rebuild the files (rewriting them in the process)
```

Comment:
This test has not survived the test of time well.

Let's just delete this test from this clause down, but leave the above clauses. We should have tons of tests for rebuilding, so it's not critical that this exact tests works.

Also note that this test is in no way testing what its name implies as well :)